Python/DepartmentObj.py

```diff
@@ -0,0 +1,245 @@
+import Algorithm as alg
+
+# This is the list of all valid department names
+ValidDepartments = ["BAKERY",\
```

Comment:
Not sure if it is necessary, but we could probably move these strings into a resource file.

Reply:
I created a define file and moved the strings there.